ocp_resources/cluster_role.py

```diff
@@ -30,39 +29,12 @@ def __init__(
             delete_timeout=delete_timeout,
             **kwargs,
         )
-        self.api_groups = api_groups
-        self.permissions_to_resources = permissions_to_resources
-        self.verbs = verbs
+        self.rules = rules
```

Comment:
I guess rules arg is mandatory, please raise if not passed by the user (and not yaml_file)

Reply:
ack.